.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "pytket/docs/qui-sphinx"]
+	path = pytket/docs/qui-sphinx
+	url = https://github.com/aidanCQ/qui-sphinx.git
```

Comment:
Can we move this into CQCL?

Reply:
Done.

Reply:
> Done.

Thanks! Should we control the version or commit hash that we want to use for building the pytket docs?

Reply:
Yeah, I'll see if I can add release tags

Reply:
As far as I can tell, the commit hash can be retrieved using `git submodule status`. And then when you want to update just checkout the latest commit in the submodule or git pull.

Reply:
Should we pin the docs build to use a specific tag or commit? Currently it will always use the latest version from main, I think.

Reply:
I don't think it's possible to pin to a specific commit based on some text in a file. As far as I can tell, you just checkout the commit in the submodule.

Reply:
As far as I know git will pin the submodule to the commit that is checked out. I don't think there is a way to specify this on file. In the github file browser you can see the pinned commit in the folder name: https://github.com/aidanCQ/tket/tree/feature/docs_navbar-patch/pytket/docs